modules/network-observability-operator-release-notes-1-10-technology-preview-features.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-release-notes-1-10.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-operator-release-notes-1-10-technology-preview-features_{context}"]
+= Network Observability Operator 1.10 Technology Preview features
+
+[id="network-observability-operator-custom-alerts-technology-preview_{context}"]
+== Network Observability Operator custom alerts (Technology Preview)
+This release introduces new alert functionality, and custom alert configuration using Prometheus Query Language (`PromQL`). These capabilities are Technology Preview features.
```

Comment:
Suggestion: should we clarify that it's not enabled by default? Or that's implicit as a TP.. It's also covered in your other PR, so I don't know...

```suggestion
This release introduces new alert functionality, and custom alert configuration using Prometheus Query Language (`PromQL`). These capabilities are Technology Preview features and need be enabled explicitly.
```